code/navigation.py

```diff
@@ -0,0 +1,24 @@
+# This file contains interfaces for navigation actions that can typically be executed within an editor. 
+# These interfaces can then be implemented per editor and called directly from .talon files by 
+# prefixing the method name with "user." and adding "tag: user.navigation_commands".
+
+from talon import ctrl, ui, Module, Context, actions, clip
+mod = Module()
+mod.tag(
+    "navigation_commands",
+    desc="Tag for enabling generic editor commands",
```

Comment:
we probably want a more specific description, maybe something as simple as "Tag for enabling generic editor *cursor navigation* commands"?

Reply:
copy paste mistake, fixed